docs/modules/sources/pages/csv.adoc

```diff
@@ -0,0 +1,40 @@
+= CSV Source
+
+The `xls` source can also create a requisition from a UTF-8 encoded comma separated variable (CSV) file if the given file name has the `.csv` suffix.
+
+In this case the first line of the csv file should contain a comma separated list of headers in the same way the spreadsheet should be configured in a normal xls source. 
+Alternatively, a property can be set containing comma separated set of headers to be is used as the first line of the csv file.
+This allows csv file to be raad which don't have a header line.
+
+[options="header",autowidth"]
+|===
+| Parameter         | Required | Description
+| `source`          | *        | set `xls` to use the XLS source for this requisition
+| `source.file`     | *        | path of the CSV file to read relative to the `requisition.properties`.Note the file name must end with `.csv`
+| `source.org.opennms.pris.spreadsheet.fields` |          | If this property is not present, the first line of the csv file is used as the column headers in the same way as in the xls data source. 
+If this property is set, value of this field should be a comma separated set of headers to be used instead of using the first line of the csv file.
```

Comment:
```suggestion
If this property is set, the value of this field should be a comma-separated set of headers to use instead of using the first line of the CSV file.
```